erpnext/manufacturing/doctype/production_planning_tool/production_planning_tool.json

```diff
@@ -646,6 +646,33 @@
    "search_index": 0, 
    "set_only_once": 0, 
    "unique": 0
+  },
+  {
+   "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "default": "1", 
+   "depends_on": "create_material_requests", 
+   "description": "If checked, Projected qty will be ignored. If material requests are generated based on sales orders, the qty reserved by sales orders will be included in projected qty, which causes the qty deduction of the material requests. Thus, ignorance of projected qty should be checked in this situation.", 
+   "fieldname": "ignore_projected_qty", 
+   "fieldtype": "Check", 
+   "hidden": 0, 
+   "ignore_user_permissions": 0, 
+   "ignore_xss_filter": 0, 
+   "in_filter": 0, 
+   "in_list_view": 0, 
+   "label": "Ignore Projected Qty", 
```

Comment:
"ignore_projected_qty" has been changed to "create_material_requests_for_all_required_qty".